config/initializers/rails_settings.rb

```diff
@@ -0,0 +1,57 @@
+require "request_store"
```

Comment:
if we decide to lean on `RequestStore` upstream, we need to promote it as a full on dependency in the `Gemfile`

ps. how do we make sure this monkey patch doesn't break each time the gem is updated? Do we have a test?

Reply:
@rhymes I updated the monkey-patch to use `ActiveSupport::CurrentAttributes` instead. Could you maybe give this another spin?

> ps. how do we make sure this monkey patch doesn't break each time the gem is updated? Do we have a test?

The easiest way would be pinning the dependency to an exact version until we decided to completely take it over, which is what I did now. In general there are quite a few tests that cover that this works, as I found out before it worked 😉 

I also started on our own version that will live in `Settings::Base` but I'd rather finish this at the end once all tables have been migrated.